assets/js/_main.js

```diff
@@ -84,7 +84,7 @@ $(document).ready(function() {
   // add lightbox class to all image links
   $(
     "a[href$='.jpg'],a[href$='.jpeg'],a[href$='.JPG'],a[href$='.png'],a[href$='.gif'],a[href$='.webp']"
-  ).addClass("image-popup");
+  ).has("> img").addClass("image-popup");;
```

Comment:
@mmistakes I accidentally left an extra semicolon here. Would you mind helping remove it? I'm asking because I wouldn't bother with another PR for this thing too trivial (and it really changes nothing functional).

No rebuild is needed as uglify.js is smart enough to ignore these semicolons and turn them into commas.